css/cssom-view/scrollIntoView-vertical-rl-writing-mode.html

```diff
@@ -68,8 +68,20 @@
 // As browsers differ in the meaning of scrollLeft when
 // in a right-to-left mode, we adjust the expectation
 // to match the semantics of scrollLeft.
-if(scroller.scrollLeft === 0)
-  expectedX = [ -box_width, -(((3*box_width - scroller_width)/2)+ (scrollbar_width/2)), -(((2*box_width)-scroller_width)+scrollbar_width)];
+// As browsers differ in the meaning of scrollLeft when
+// in a right-to-left mode, we adjust the expectation
+// to match the semantics of scrollLeft.
+if(scroller.scrollLeft === 0) {
+  // If scroll bar is on the left side, scroller.scrollLeft won't bigger then 0,
```

Comment:
"If vertical scrollbar is on the right side, one can scrollLeft to positive values in order to show the content hidden by the bar?"

Reply:
Yes.